src/main/java/mate/academy/hibernate/relations/Main.java

```diff
@@ -15,18 +21,24 @@ public static void main(String[] args) {
         // use this session factory when you will initialize service instances
         SessionFactory sessionFactory = HibernateUtil.getSessionFactory();
 
+        CountryDaoImpl countryDao = new CountryDaoImpl(sessionFactory);
         Country usa = new Country("USA");
-        CountryService countryService = null; // TODO: initialize this instance
+        CountryService countryService =
+                new CountryServiceImpl(countryDao); // TODO: initialize this instance
         countryService.add(usa);
 
+        ActorDaoImpl actorDao = new ActorDaoImpl(sessionFactory);
         Actor vinDiesel = new Actor("Vin Diesel");
         vinDiesel.setCountry(usa);
-        ActorService actorService = null; // TODO: initialize this instance
+        ActorService actorService =
+                new ActorServiceImpl(actorDao); // TODO: initialize this instance
         actorService.add(vinDiesel);
 
+        MovieDaoImpl movieDao = new MovieDaoImpl(sessionFactory);
         Movie fastAndFurious = new Movie("Fast and Furious");
         fastAndFurious.setActors(List.of(vinDiesel));
-        MovieService movieService = null; // TODO: initialize this instance
+        MovieService movieService =
+                new MovieServiceImpl(movieDao); // TODO: initialize this instance
         movieService.add(fastAndFurious);
         System.out.println(movieService.get(fastAndFurious.getId()));
```

Comment:
It is not a good practice to print information directly in the main method. Consider logging this information or handling it in a different part of your application.